### PR TITLE
Add iPhone 11 support

### PR DESCRIPTION
**Pull request by tulusheg**

### CODE REVIEW

File: GBDeviceInfo/GBDeviceInfo_iOS.m [hunk 1]
```diff
@@ -224,6 +224,15 @@ + (NSArray *)_modelNuances {
...
                 @[@11, @6]: @[@(GBDeviceModeliPhoneXSMax), @"iPhone XS Max", @(GBDeviceDisplay6p5Inch), @458],
+                
+                // 11
+                @[@12, @1]: @[@(GBDeviceModeliPhone11), @"iPhone 11", @(GBDeviceDisplay6p1Inch), @326],
+
+                // 11 Pro
+                @[@12, @3]: @[@(GBDeviceModeliPhone11Pro), @"iPhone 11 Pro", @(GBDeviceDisplay5p8Inch), @458],
+
+                // 11 Pro Max
+                @[@11, @5]: @[@(GBDeviceModeliPhone11ProMax), @"iPhone 11 Pro Max", @(GBDeviceDisplay6p5Inch), @458],
```
Comment (WesleyJacobsMP) [inline GBDeviceInfo/GBDeviceInfo_iOS.m]: Shouldn't this be `@[@12, @5]`?

Comment (tulusheg) [reply GBDeviceInfo/GBDeviceInfo_iOS.m]: Right, thanks. Fixed it.

**Comment by WesleyJacobsMP** [comment]
@lmirosevic Can you get this out? Would be really nice to have this asap :)

**Comment by lmirosevic** [comment]
@tulusheg Thank you for the PR! Merged. I will cut a new release shortly.

**Pull request merged**